### PR TITLE
Fixed wrong behavior with logout handler.

### PR DESCRIPTION
**Pull request by jvelilla**
Updated request util class, to unset a current.
Workaround cms_api to use sqlite storage. (only for testing purposes).

### CODE REVIEW

File: src/service/cms_api.e [hunk 1]
```diff
@@ -36,11 +36,26 @@ feature {NONE} -- Initialize
 			to_implement ("Refactor database setup")
 			if attached setup.storage (error_handler) as l_storage then
 				storage := l_storage
+			elseif attached sqlite_storage as l_sqlite then
```
Comment (jocelyn) [inline src/service/cms_api.e]: I am not sure why you need that?

Comment (jvelilla) [inline src/service/cms_api.e]: in other case the storage is always CMS_STORAGE_NULL, it's a workaround to make the example works.

Comment (jocelyn) [inline src/service/cms_api.e]: but it works only if the user install the sqlite odbc driver.

Comment (jvelilla) [inline src/service/cms_api.e]: I agree

**Pull request closed without merging**